WordPress/Classes/ViewRelated/Me/MeHeaderView.h

```diff
@@ -1,18 +1,20 @@
 #import <UIKit/UIKit.h>
 
 typedef void (^MeHeaderViewCallback)(void);
+typedef void (^MeHeaderViewDropCallback)(UIImage * _Nonnull image);
 
 @interface MeHeaderView : UIView
 
-@property (nonatomic,   copy) NSString              *displayName;
-@property (nonatomic,   copy) NSString              *username;
-@property (nonatomic,   copy) NSString              *gravatarEmail;
-@property (nonatomic,   copy) MeHeaderViewCallback  onGravatarPress;
-@property (nonatomic, assign) BOOL                  showsActivityIndicator;
+@property (nonatomic, nonnull,  copy) NSString                  *displayName;
```

Comment:
`displayName` might need to be `nullable` or we might need to edit `MeViewController headerViewForAccount` to check if `account.displayName` is `nil`.  
The reason is when a user first logs in account information has not fully synced.  In this scenario the value of `account.displayName` is `nil` when `MeHeaderView` is first updated and this change will result in a crash.

Reply:
Okay. I wasn't aware that the `displayName` could ever be `nil`. Fixed